src/app/righthand-panel.js

```diff
@@ -118,7 +121,9 @@ function RighthandPanel (container, appAPI, events, opts) {
   })
 
   if (appAPI.config.exists(EDITOR_WINDOW_SIZE)) {
-    self._api.setEditorSize(appAPI.config.get(EDITOR_WINDOW_SIZE))
+    setTimeout(function () {
```

Comment:
i think the whole layout and collapsing and drag resizing system is a little bit broken and needs and overhaul. There seems to be a lot of code (way more then necessary) in often half random positions to make it work.

In the big refactor pull request i already did one more serious refactoring, but I think it can even be better than that. So maybe we could open an issue for that to make all of this more nice in the future